app/services/PsaSchemeDashboardService.scala

```diff
@@ -62,6 +65,16 @@ class PsaSchemeDashboardService @Inject()(
     val linkText = if (!isSchemeOpen(ua)) {
       viewLinkText
     } else {
+
+      logger.warn(s"Pension-scheme : $srn -- Lock-Status : ${lock.getOrElse("No-Lock-Found").toString}")
+
+      ua.get(ListOfPSADetailsId).map {
+        listOfPSADetails =>
+          listOfPSADetails.map { psaDetails =>
+            logger.warn(s"Pension-scheme : $srn -- PsaDetails-ID : ${psaDetails.id} -- PsaDetails-FullName : ${psaDetails.getPsaName}")
```

Comment:
I don't think we're allowed to put personally identifiable info in the kibana logs, so remove name. PSA ID may be OK